pytype/tests/py3/test_annotated.py

```diff
@@ -0,0 +1,65 @@
+"""Tests for PEP-593 typing.Annotated types."""
+
+from pytype.tests import test_base
+
+
+class AnnotatedTest(test_base.TargetPython3BasicTest):
```

Comment:
This test class should inherit from TargetPython3*Feature*Test. For the GitHub tests it doesn't matter, but internally, TargetPython3BasicTest tests are run in both Python 3 and Python 2 (using a patched interpreter that has type annotations backported).

Reply:
Done, thanks for explaining the difference!